src/app/exercise/exercises/TriadInversionExercise/triadInversionExercise.ts

```diff
@@ -58,19 +59,38 @@ export const triadInversionExercise = () => {
       const invertionOptions: TriadInversion[] = [0, 1, 2].filter(invertionOption => settings.includedAnswers.includes(triadInversions[invertionOption]));
       const randomTriadInversion: TriadInversion = randomFromList(invertionOptions);
       const answer = triadInversions[randomTriadInversion];
-      const voicing: Note[] = new Chord(randomChordInC).getVoicing({
+      let voicing: Note[] = new Chord(randomChordInC).getVoicing({
         topVoicesInversion: randomTriadInversion,
         withBass: false,
         octave: 3, // picking a lower octave as a high one is more difficult
       });
+      const root: Note = voicing[(3 - randomTriadInversion) % 3];
+      switch (settings.arpeggioDirection) {
+        case 'descending':
+          voicing = voicing.reverse();
+          break;
+        case 'ascendingAndDescending':
+          voicing = voicing.concat([...voicing].reverse());
+          break;
+        case 'descendingAndAscending':
+          voicing = voicing.reverse().concat([...voicing].reverse());
+          break;
+      }
       const question: Exercise.Question<TriadInversionAnswer> = {
         segments: [
           {
             partToPlay: voicing.map((note, index) => {
               const noteDelay = index * settings.arpeggiateSpeed / 100;
+              let velocity = 0.3;
+              if ((settings.arpeggioDirection === 'ascendingAndDescending' || 
+                   settings.arpeggioDirection === 'descendingAndAscending') &&
+                   settings.arpeggiateSpeed === 0) //playing whole note at the same time is too loud
+              {
+                velocity /= 2;
+              }
```

Comment:
Instead of this, let's just not apply the direction code in case `arpeggiateSpeed === 0`. I.e. add an `if (arpeggiateSpeed !== 0)` statement surrounding the switch at line 68.